tests/sentry/snuba/test_discover.py

```diff
@@ -349,6 +349,39 @@ def test_selected_columns_aggregate_alias(self, mock_query):
             referrer=None,
         )
 
+    @patch("sentry.snuba.discover.raw_query")
+    def test_selected_columns_aggregate_alias_with_brackets(self, mock_query):
+        mock_query.return_value = {
+            "meta": [{"name": "transaction"}, {"name": "p95()"}],
+            "data": [{"transaction": "api.do_things", "p95()": 200}],
```

Comment:
Should this be `p95` without parens?

Reply:
If I were concerned about exactly matching what the backend returns then right now yes. Of course we don't use this value so it doesn't matter right now.